internal/workflow_shadower.go

```diff
@@ -338,10 +338,9 @@ func (o *ShadowOptions) validateAndPopulateFields() error {
 			}
 			statuses = append(statuses, status)
 		}
-		//All the open statuses are taken by default. This list seems to not work as expected.
-		//TODO: verify that the status list works as expected. currently all wfs of all types get picked up.
+		//This filter doesn't seem to be working as expected.
```

Comment:
please elaborate in the comment what you mean

Reply:
This filter should have only taken open and closed wfs but it returns continue as new wfs as well. not sure if this is because of some other check somewhere or the query gets formed incorrectly.